index.js

```diff
@@ -0,0 +1,52 @@
+const title = document.querySelector('.title_input');
+const author = document.querySelector('.author_input');
+const empty = document.querySelector('.empty_bookstore');
+const form = document.querySelector('.book_form');
+const tableBody = document.querySelector('.table_body');
+const storedBooks = JSON.parse(localStorage.getItem('bookstored'));
+
+function displayBooks(booksData) {
+  let eachTableRow = '';
+  for (let i = 0; i < booksData.length; i += 1) {
+    eachTableRow
+      += `<tr>
+            <td>${booksData[i].titleKey}</td>
+            <td>${booksData[i].authorKey}</td>
+            <td><button type="button" id=${i} class="remove_btn">Remove</button></td>
+        </tr>
+        `;
+  }
+  tableBody.innerHTML = eachTableRow;
+  const deleteBookBtn = document.querySelectorAll('.remove_btn');
+  deleteBookBtn.forEach((elementBtn) => {
+    elementBtn.addEventListener('click', () => {
+      const index = elementBtn.getAttribute('id');
+      storedBooks.splice(index, 1);
+      /* eslint-disable */
+      localStorage.clear;
+      /* eslint-enable */
```

Comment:
- [ ] **[OPTIONAL]** Do we really need to clear the local storage every time? The `setItem()` typically method overrides whatever exists in the `storedBooks` variable so there is no need to clear the local storage.